docs/content/tutorials/virtual-server-with-custom-listener-ports.md

```diff
@@ -0,0 +1,171 @@
+---
+title: VirtualServer with Custom listener ports
+description: |
+  How to configure a VirtualServer with custom HTTP and HTTPS listener ports.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+---
+## Configuring a VirtualServer with custom HTTP and HTTPS listener ports.
+
+This tutorial outlines how to configure and deploy a VirtualServer resource to with custom HTTP and HTTPS listener ports.
+Previously this was task had to be performed manually as documented in [Customizing NGINX Ingress Controller Ports](/nginx-ingress-controller/tutorials/custom-listen-ports/).
```

Comment:
```suggestion
This task can be performed manually as documented in ["Customizing NGINX Ingress Controller Ports"](/nginx-ingress-controller/tutorials/custom-listen-ports/): in this tutorial, we will explain how to do so automatically as part of a deployment.
```

Documentation is evergreen: we tend to avoid anything in writing that has any narrative involving the passage of time. Even if this is newer functionality we don't draw attention to that fact outside of release notes and similar changelogs.